src/cider/nrepl/middleware/out.clj

```diff
@@ -0,0 +1,86 @@
+(ns cider.nrepl.middleware.out
+  "Change *out* to print on connections in addition to process out.
+
+  Automatically changes the root binding of *out* to print to any
+  active connections. An active connection is a session that has sent
```

Comment:
Mixing up connection and session is a bit confusing IMO. Maybe we should talk only about sessions here.